trackmania.js

```diff
@@ -287,7 +287,7 @@ export class CoreService extends BaseNadeoService {
      * @param {*} seasonId 
      * @returns {Promise<any[]>}
      */
-    async getMapRecords(accountIdList, mapId, gameMode, seasonId = undefined) {
+    getMapRecords = async (accountIdList, mapId, gameMode, seasonId = undefined) => {
```

Comment:
Wat

Reply:
[https://stackoverflow.com/questions/71094583/cannot-read-properties-of-undefined-javascript-class](url)
i had this problem